doc/source/cluster/launcher-usage.rst

```diff
@@ -72,6 +72,8 @@ You can also run ``ray up`` to restart a cluster if it seems to be in a bad stat
 
 If you don't want the update to restart services (e.g., because the changes don't require a restart), pass ``--no-restart`` to the update call.
 
+If you want to re-apply a previously applied cluster configuration, pass ``--no-config-cache`` to the update call.
```

Comment:
```suggestion
If you want to force re-generation of the config to pick up possible changes in the Cloud environment, pass ``--no-config-cache`` to the update call.
```

Reply:
Minor/pedantic: Is the capitalization of "Cloud" intentional?